creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -127,17 +127,17 @@ class PeerSetManager {
     try {
       // Request all users that have this node as a replica (either primary or secondary)
       const resp = await Utils.asyncRetry({
+        asyncFnLabel: 'fetch all users with this node in replica',
         asyncFn: axios,
-        asyncParams: {
+        options: {
```

Comment:
`options` is actually options to pass into the async-retry npm package. you should actually pass in the fn into `asyncFn`. 

[example](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/test/utils.test.js#L110-L125)

Reply:
Oh gotcha - that example definitely helped